packages/typescriptlang-org/src/templates/tsconfigReference.tsx

```diff
@@ -74,7 +74,7 @@ const TSConfigReferenceTemplateComponent = (props) => {
 
   return (
     <Layout title={i("tsconfig_title")} description={i("tsconfig_description")} lang={props.pageContext.locale} allSitePage={props.data.allSitePage}>
-      <div className="tsconfig raised main-content-block">
+      <div className="tsconfig raised main-content-block markdown">
```

Comment:
[The algolia indexer looks for `.markdown hX`](https://github.com/algolia/docsearch-configs/blob/34df0a0c50a3395d589ff2df225f3677e09c0250/configs/typescriptlang.json#L42)